PowerUp/app/src/main/java/powerup/systers/com/ui/scenario_over_screen/ScenarioOverActivity.java

```diff
@@ -28,25 +28,34 @@
 import android.widget.LinearLayout;
 import android.widget.TextView;
 
-import powerup.systers.com.datamodel.Scenario;
-import powerup.systers.com.datamodel.SessionHistory;
-import powerup.systers.com.db.DatabaseHandler;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import powerup.systers.com.ui.GameOverActivity;
+import powerup.systers.com.R;
+import powerup.systers.com.data.DataSource;
+import powerup.systers.com.data.entities.Scenario;
+import powerup.systers.com.data.SessionHistory;
+import powerup.systers.com.ui.game_activity.GameActivity;
+import powerup.systers.com.ui.map_screen.MapActivity;
+import powerup.systers.com.utils.InjectionClass;
 import powerup.systers.com.utils.PowerUpUtils;
 
 
-public class ScenarioOverActivity extends AppCompatActivity {
+public class ScenarioOverActivity extends AppCompatActivity implements ScenarioOverContract.IScenarioOverView{
 
     public Activity scenarioOverActivityInstance;
     public static int scenarioActivityDone;
-    private DatabaseHandler mDbHandler;
-    public Scenario scene;
+    private DataSource dataSource;
+    public Scenario scene, prevScene;
+
+    //Todo check preferences and shift it to pref file
```

Comment:
For this Todo item, are you going to address it through this PR itself?

Reply:
I thought to create first-timer issue for it as App works fine now. If we can move it pref file instead, code will be cleaner.